pkg/cloud/gcp/actuators/machine/reconciler_test.go

```diff
@@ -122,6 +122,33 @@ func TestCreate(t *testing.T) {
 				}
 			},
 		},
+		{
+			name: "guestAccelerators are correctly passed to the api",
+			providerSpec: &gcpv1beta1.GCPMachineProviderSpec{
+				Region:      "test-region",
+				Zone:        "test-zone",
+				MachineType: "n1-test-machineType",
+				GuestAccelerators: []*gcpv1beta1.GCPAcceleratorConfig{
+					{
+						AcceleratorType:  "nvidia-tesla-v100",
+						AcceleratorCount: 2,
+					},
+				},
+			},
+			validateInstance: func(t *testing.T, instance *compute.Instance) {
+				if len(instance.GuestAccelerators) != 1 {
```

Comment:
2 questions,
1. is this `validateInstance` mocking the gcp api?
2. should we have a test for a failure/error response from `validateInstance`?

Reply:
ack, thanks!